### PR TITLE
Added support for java.util.Date SwissKnife...

### PR DESCRIPTION
**Pull request by pedro-stanaka**
In order to allow the BodyParserEnginePost to parse POJO's with fields of Date type.

### CODE REVIEW

**Comment by gitblit** [comment]
I think this is a good start for getting Date support but the [docs](http://www.ninjaframework.org/documentation/basic_concepts/argument_extractors.html) must be updated to explain the [use and formatting rules](http://joda-time.sourceforge.net/api-release/org/joda/time/format/ISODateTimeFormat.html#localDateOptionalTimeParser%28%29).  I'm on the fence about custom formats and non-local timezones; sometimes [KISS](http://en.wikipedia.org/wiki/KISS_principle) is better.

**Comment by raphaelbauer** [comment]
@pedro-stanaka This looks like a very good start! @gitblit What do you think is missing?

**Comment by pedro-stanaka** [comment]
I've updated the docs. Please check if anything is wrong.

What about create a new session about POJO parsing on the docs? I could create it, I _kinda_ missed It when reading the docs. What about cover the following topics:
- [ ] General usage.
- [ ] Data types supported;
- [ ] The name of fields should match the POJO fields (case sensitive, am I right?)

File: ninja-core/src/main/java/ninja/utils/SwissKnife.java [hunk 1]
```diff
@@ -203,6 +203,14 @@ public static Short toShort(String value) {
             return Short.valueOf(value);
         }
 
+        public static Date toDate(String value){
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/utils/SwissKnife.java]: really picky... but could you please add spaces between the brackets? ) {, and if (... and  } else and so on? This uniformity helps us to keep the codebase maintainable in the long run :)

**Comment by raphaelbauer** [comment]
@pedro-stanaka I think that PR is really nice (Did you al:) @gitblit Do you have any comments? Otherwise I'd merge it asap..

Please make sure you also signed the contributor's agreement and added yourself to team.md (See http://www.ninjaframework.org/developer/getting_started.html).

Regarding extension of other parts of the documentation: Very good idea! Simply sketch some ideas and let's discuss them on github or on the mailing list :)

**Comment by pedro-stanaka** [comment]
@raphaelbauer I've done everything that is asked/advised to do in this getting started page. And I have no more comments, so I think you can merge it!! :) 

P.S. I've added the whitespaces. Thanks for the tip.

**Comment by gitblit** [comment]
:thumbsup:  I vote to merge it.

**Comment by pedro-stanaka** [comment]
Just asking here.... When there will be a new release of the framework? I'm going to teach a course on Java Web, and I'm going to use Ninja on it. 
And I really need this date feature working for the course. Is there a way for me to make custom build?

**Comment by raphaelbauer** [comment]
Hey pedro-stanaka,

you can always checkout this repo and "mvn clean install" to use the snapshot version locally...
In general there is a Ninja release each month or so...

Best,

Raphael

**Comment by pedro-stanaka** [comment]
Thanks for everything, I will look forward to continuing on contribute to the repo.

**Pull request merged**